src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const basePrice = days * PRICE_PER_DAY;
+
+  // Логика скидок
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT; // Скидка $50 для 7 или более дней
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return basePrice - SHORT_TERM_DISCOUNT; // Скидка $20 для 3–6 дней
```

Comment:
Similarly, the discount logic for 3 to 6 days applies a flat $20 discount. Ensure this aligns with the task requirements. If a different discount method is required, such as a percentage or per-day discount, this should be modified.